src/services/fileSystem.ts

```diff
@@ -25,19 +27,19 @@ export const deleteStoredWorkpass = () => {
   return FileSystem.deleteAsync(DEFAULT_FILE_NAME);
 };
 
-const TIME_ACCEPTED_FILE_NAME = "@storedTimeAccepted";
-
-export const setTimeAccepted = time => {
-  return FileSystem.writeAsStringAsync(TIME_ACCEPTED_FILE_NAME, time);
+export const storeTime = async () => {
+  await AsyncStorage.setItem("@storedTimeAccepted", getCurrentDateAndTime());
 };
 
-export const getTimeAccepted = () => {
-  return FileSystem.readAsStringAsync(TIME_ACCEPTED_FILE_NAME);
+export const getStoredTime = async () => {
+  const storedTimeAccepted = await AsyncStorage.getItem("@storedTimeAccepted");
```

Comment:
```suggestion
  return AsyncStorage.getItem("@storedTimeAccepted");
```

You can even remove the `async` in the function signature. 
`return await fn()` is the same as `return fn()`